.github/workflows/test-vurnabilities-data.yml

```
@@ -0,0 +1,124 @@
+                const txtFilePath = path.join(process.env.GITHUB_WORKSPACE, 'vulnerability_base_data.txt');
+                fs.writeFileSync(txtFilePath, txtContent);
+                console.log(`Data successfully written to ${txtFilePath}`);
+
+                client.release();
+                return extractedData; // Return the extracted data
+              } catch (err) {
+                console.error('Error fetching vulnerability data:', err);
+                client.release();
+              }
+            })();
+              
```

yamllint

[error] trailing spaces

(trailing-spaces)